Makefile

```diff
@@ -4,6 +4,10 @@ comma := ,
 help = help::; @echo $$$$(tput bold)$(strip $(1)):$$$$(tput sgr0) $(strip $(2))
 $(call help,make help,print this help message)
 
+.PHONY: services
+
+.PHONY: devdata
```

Comment:
Do-nothing `make services` and `make devdata` commands got added to projects that don't have services or devdata. This makes scripting easier because you can always run `make services devdata` and it won't crash